larq/layers_base.py

```diff
@@ -3,7 +3,8 @@
 import tensorflow as tf
 
 from larq import metrics as lq_metrics
-from larq import quantized_scope, quantized_variable, quantizers
+from larq import quantized_scope, quantizers
```

Comment:
I think it'd be cleaner to combine these `from larq` imports:

```python
from larq import quantized_scope, quantizers, metrics as lq_metrics
```

Reply:
I agree, but `isort` doesn't 😉

Reply:
Changed the config in 9381fe27fbfb9ba79b3c0e8187401513944ef695